src/multirust-cli/download_tracker.rs

```diff
@@ -144,7 +145,7 @@ impl DownloadTracker {
                 } else {
                     Cow::Borrowed("Unknown")
                 };
-                let _ = write!(&mut self.term,
+                let _ = write!(self.term.as_mut().unwrap(),
```

Comment:
No. There is a branch in `handle_notification` that short-circuits most of the code in this module.